ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -102,13 +102,15 @@ def main(argv=sys.argv[1:]):
     report = {}
     for filename in files:
         report[filename] = []
-    for error in root.findall('error'):
-        filename = error.get('file')
+    errors = root.find('errors')
+    for error in errors:
```

Comment:
fixed in https://github.com/ament/ament_lint/pull/69/commits/c279be12d5366ee1045e2276a2c5e6900a71d677